PubnativeLite/PubnativeLite/VAST/PNLiteVASTPlayerViewController.m

```diff
@@ -591,48 +636,19 @@ - (void)setLoadState {
     self.wantsToPlay = NO;
     [self.loadingSpin startAnimating];
     
-    if (!self.vastUrl && !self.vastString) {
```

Comment:
Those changes (all of the changes in `setLoadState` method) have to be reverted back i think since they are dependent on videoAdCache. I forgot to add a case in development branch as well. But since we won't have VideoAdCache in IQV, reverting this will be fine. I have to make a change in the development branch.

Reply:
already reverted.